backend/globaleaks/handlers/accreditator/__init__.py

```diff
@@ -0,0 +1,158 @@
+from globaleaks.handlers.accreditator.services import accreditation, get_all_accreditation, persistent_drop, \
+    get_accreditation_by_id, update_accreditation_by_id, toggle_status_activate, activate_tenant, \
+    from_invited_to_request
+from globaleaks.handlers.base import BaseHandler
+from globaleaks.rest import requests, errors
+import logging
+import json
+
+class ConfirmRequestHandler(BaseHandler):
+    """
+    This manager is responsible for receiving accreditation requests and forwarding them to the accreditation manager
+    """
+    check_roles = 'any'
```

Comment:
SECURITY ISSUE: any authenticated user can access this handler regardless of their permissiona